apps/sscs/sscs-tribunals-frontend/sscs-tribunals-frontend.yaml

```diff
@@ -7,6 +7,11 @@ spec:
   releaseName: sscs-tribunals-frontend
   values:
     nodejs:
+      autoscaling:
+        enabled: true
+        maxReplicas: 4
+      memoryRequests: '512Mi'
+      memoryLimits: "1024Mi"
```

Comment:
cpu requests should be defined and increased from the nodejs default of 25m. Current utilisation in prod is ~31m, so this would cause the HPA to scale to max and stay there indefinitely. Suggested value for cpuRequests = 100m.